offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/create/CreateOfferPresenter.groovy

```diff
@@ -50,4 +51,8 @@ class CreateOfferPresenter implements CreateOfferOutput {
        this.viewModel.failureNotifications.add(notification)
     }
 
+    @Override
+    void fetchedOffer(Offer fetchedOffer) {
+        this.createOfferViewModel.savedOffer = Optional.ofNullable(fetchedOffer)
```

Comment:
```suggestion
        this.createOfferViewModel.savedOffer = Optional.of(fetchedOffer)
```

Reply:
Fail fast principle.

Reply:
I would also expect that the use case won't proxy a null object reference, that would be really bad.